src/main/Duke.java

```diff
@@ -0,0 +1,97 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Duke {
+    private static boolean is_bye = false;
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    public enum Commands {
+        bye,
+        list,
+        done,
+        deadline,
+        
+    }
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Scanner scan = new Scanner(System.in);
+
+        while(!is_bye){
```

Comment:
Your logic is easy to understand and seems to work well, great job! However, do you think it would be tidier and more readable to use switch statements instead of many if-else statements?